doc/quickstart/commandline_quickstart.rst

```diff
@@ -69,8 +69,8 @@ cd /home/user goes to your home directory
 same as
 cd ~
 
-`cd ..` moves you one directory to the top 
-`cd user` goes from your current location to the home directory of user. This is not the same as `/user`
+`cd ..` moves you one directory to the top, 
+`cd ~user` goes from your current location to the home directory of user. This is not the same as `/user`
```

Comment:
`cd ` 

Please change to `cd `  -> without anything cd will go to the home directory

Reply:
Dear Astrid, the back quotes take me into the wrong direction. I should also delete "user". The right line should be:
`cd` goes from your current location to the home directory .....
I don't know how to edit files in the pull request. Should I send a new pull request?

Reply:
Hello @zsiki, you are right. 
you can go in the browser to https://github.com/zsiki/OSGeoLive-doc to your branch and edit the file there.